src/Plugins/ComplexCore/src/ComplexCore/Filters/Algorithms/ConvertColorToGrayScale.cpp

```diff
@@ -138,6 +138,7 @@ class ParallelWrapper
   {
     ParallelDataAlgorithm dataAlg;
     dataAlg.setRange(0, totalPoints);
+    dataAlg.setParallelizationEnabled(false);
```

Comment:
Check if parallelization disabled is intended.

Reply:
@mmarineBlueQuartz Can we check if the DataArray is out-of-core then turn off parallel otherwise leave it on?